plugins/main/public/controllers/overview/components/last-alerts-stat/last-alerts-stat.tsx

```diff
@@ -155,7 +155,7 @@ export function LastAlertsStat({
                 }}
                 href={discoverLocation}
               >
-                {statValue}
+                {Number(statValue).toLocaleString('en-US')}
```

Comment:
This should be done specifically when statValue is a number. 

I think that's why the snapshot is updated to NaN.